Parsers/ASimUserManagement/Parsers/vimUserManagementCiscoISE.yaml

```diff
@@ -0,0 +1,175 @@
+            and (array_length(actorusername_has_any) == 0 or SyslogMessage has_any (actorusername_has_any))
+        //***************************** </Prefiltering> *************************
+        | where ProcessName has_any ("CISE", "CSCO")
+        | parse SyslogMessage with * " " longvalue:long " " EventOriginalType:int " " *
+        | where EventOriginalType in (EventOriginalTypeList)
+        | project
+            TimeGenerated,
+            EventTime,
+            EventOriginalType,
+            Computer,
+            SyslogMessage,
+            HostName,
+            HostIP
+        | lookup EventFieldsLookup on EventOriginalType
+        | parse-kv SyslogMessage as (NetworkDeviceName: string, ['User-Name']: string, UserName: string, User: string, ['Remote-Address']: string) with (pair_delimiter=',', kv_delimiter='=')
+        | project-rename
+            SrcIpAddr=['Remote-Address']
+        | where (array_length(srcipaddr_has_any_prefix) == 0 or has_any_ipv4_prefix(SrcIpAddr, srcipaddr_has_any_prefix))
+        | extend dvcHostname = coalesce(NetworkDeviceName, Computer, HostName)
+        | extend ActorUsername = coalesce(['User-Name'], UserName, User)
+        | extend ActorUsernameType = _ASIM_GetUsernameType(ActorUsername)
+        | where (array_length(actorusername_has_any) == 0 or ActorUsername has_any (actorusername_has_any))     
+        | extend
+            DvcIpAddr = iif(isnotempty(HostIP) and HostIP != "Unknown IP", HostIP, extract(@"(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})", 1, Computer))
+            , EventStartTime = coalesce(EventTime, TimeGenerated)
+            , EventEndTime = coalesce(EventTime, TimeGenerated)
+            , EventVendor = "Cisco"
+            , EventProduct = "ISE"
+            , EventProductVersion = "3.2"
+            , EventCount = int(1)
+            , EventSchema = "UserManagement"
+            , EventSchemaVersion = "0.1.1"
+        // ***************** <Aliases> ********************
+        | invoke _ASIM_ResolveDvcFQDN('dvcHostname')
+        | extend 
+            Hostname = DvcHostname
+            , IpAddr = SrcIpAddr
+            , Src = SrcIpAddr
+            , UpdatedPropertyName = EventSubType
+            , User = ActorUsername
+        // ***************** </Aliases> *******************
+        | project-away
+            Computer,
+            SyslogMessage,
+            HostIP,
+            NetworkDeviceName,
+            HostName,
+            dvcHostname,
+            ['User-Name'],
+            UserName
+    };    
+    CiscoISEUsrMgmtParser(
+        starttime = starttime,
+        endtime = endtime,
+        srcipaddr_has_any_prefix = srcipaddr_has_any_prefix,
+        eventresult = eventresult,
+        eventtype_in = eventtype_in,
+        actorusername_has_any = actorusername_has_any,
+        targetusername_has_any = targetusername_has_any,
+        disabled=disabled
+    )
```

Comment:
there are issues with few filters, please re-test:
eventresult filter doesn't work
targetusername_has_any declaration is missing

Reply:
fixed